src/main/resources/config.yml

```diff
@@ -39,6 +39,10 @@ refreshCommandTraders: false
 #          click the trader
 refreshCommandTradersMinutes: 5
 
+# false: wandering traders drink an invisibility potion at night
```

Comment:
please include some wording here that clarifies this option only touches wt-spawned traders